targets/wasi.json

```diff
@@ -18,5 +18,6 @@
 		"--no-demangle",
 		"{root}/lib/wasi-libc/sysroot/lib/wasm32-wasi/libc.a"
 	],
-	"emulator":      ["wasmtime"]
+	"emulator":      ["wasmtime"],
+	"wasm-abi":      "generic"
```

Comment:
I am not sure if this is necessary. It seems ok to just not set and use default empty string

Reply:
It is not necessary, but it doesn't harm either.